src/assets/i18n/en.json5

```diff
@@ -868,6 +868,10 @@
 
   "collection.form.title": "Name",
 
+  "collection.form.entityType": "Entity Type",
+
+  "collection.form.errors.entityType.required": "Please choose an entity type for this collection",
```

Comment:
As mentioned above, Entity Type should not be required for a Collection.  We need to support default Collections which can be used to create default Items.